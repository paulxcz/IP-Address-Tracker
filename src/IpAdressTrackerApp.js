import React, { useState } from "react";
import Header from "./components/Header";
import { Map } from "./components/Map";

import "../src/styles/IpAdressTrackerApp/IpAdressTrackerApp.css";
export default function IpAdressTrackerApp() {

  const [cords, setcords] = useState({
    lat: 35.63304462061077,
    lng: 139.8803621105462
  })

  return (
    <div className="Ip-adress-app">
      <Header setcords={setcords}/>

      <Map cords = {cords}/>

      <footer>Challenge by Frontend Mentor. Coded by <a href="https://github.com/paulxcz" target="_blank">Paulxcz</a>.</footer>
    </div>
  );
}
