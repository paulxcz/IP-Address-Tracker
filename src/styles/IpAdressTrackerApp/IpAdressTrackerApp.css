.Ip-adress-app {
  display: -ms-grid;
  display: grid;
  -ms-grid-rows: 256px 1fr;
      grid-template-rows: 256px 1fr;
}

.Ip-adress-app footer {
  background-color: #000;
  color: #fff;
  font-weight: bold;
  text-align: center;
  font-size: 0.8em;
}

.Ip-adress-app footer a {
  color: salmon;
}

.Ip-adress-app footer a:visited {
  color: salmon;
}

@media screen and (min-width: 1024px) {
  .Ip-adress-app footer {
    font-size: 1em;
  }
}
/*# sourceMappingURL=IpAdressTrackerApp.css.map */