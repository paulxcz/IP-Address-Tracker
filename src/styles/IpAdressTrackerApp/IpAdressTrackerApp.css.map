{
    "version": 3,
    "mappings": "AAAA,AAAA,cAAc,CAAA;EACV,OAAO,EAAE,IAAI;EACb,kBAAkB,EAAE,SAAS;CAiBhC;;AAnBD,AAII,cAJU,CAIV,MAAM,CAAA;EACF,gBAAgB,EAAE,IAAI;EACtB,KAAK,EAAE,IAAI;EACX,WAAW,EAAE,IAAI;EACjB,UAAU,EAAE,MAAM;EAClB,SAAS,EAAE,KAAK;CASnB;;AAlBL,AAWQ,cAXM,CAIV,MAAM,CAOF,CAAC,CAAA;EACG,KAAK,EAAE,MAAM;CAKhB;;AAjBT,AAcY,cAdE,CAIV,MAAM,CAOF,CAAC,AAGI,QAAQ,CAAA;EACL,KAAK,EAAE,MAAM;CAChB;;AAKb,MAAM,CAAC,MAAM,MAAM,SAAS,EAAE,MAAM;EAEhC,AACI,cADU,CACV,MAAM,CAAA;IACF,SAAS,EAAE,GAAG;GACjB",
    "sources": [
        "IpAdressTrackerApp.scss"
    ],
    "names": [],
    "file": "IpAdressTrackerApp.css"
}