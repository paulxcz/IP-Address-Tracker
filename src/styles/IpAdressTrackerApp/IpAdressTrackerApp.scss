.Ip-adress-app{
    display: grid;   
    grid-template-rows: 256px 1fr;

    footer{
        background-color: #000;
        color: #fff;
        font-weight: bold;
        text-align: center;
        font-size: 0.8em;

        a{
            color: salmon;

            &:visited{
                color: salmon;
            }
        }
    }
}

@media screen and(min-width: 1024px) {
    
    .Ip-adress-app{
        footer{
            font-size: 1em;
        }
    }
}