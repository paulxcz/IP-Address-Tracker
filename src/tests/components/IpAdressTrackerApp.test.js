import {shallow} from 'enzyme'
import IpAdressTrackerApp from '../../IpAdressTrackerApp'

describe('Pruebas ', () => {

    test('should display IpAdressTracker  correctly', () => {
        
        const wrapper = shallow(<IpAdressTrackerApp />);
        expect( wrapper ).toMatchSnapshot();
    })
  
})
