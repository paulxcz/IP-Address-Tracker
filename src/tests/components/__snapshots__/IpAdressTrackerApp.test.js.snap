// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas  should display IpAdressTracker  correctly 1`] = `
<div
  className="Ip-adress-app"
>
  <Header
    setcords={[Function]}
  />
  <Map
    cords={
      Object {
        "lat": 35.63304462061077,
        "lng": 139.8803621105462,
      }
    }
  />
  <footer>
    Challenge by Frontend Mentor. Coded by 
    <a
      href="https://github.com/paulxcz"
      target="_blank"
    >
      Paulxcz
    </a>
    .
  </footer>
</div>
`;
